app/controllers/api/v0/articles_controller.rb

```diff
@@ -1,6 +1,24 @@
 class Api::V0::ArticlesController < ApplicationController
+  before_action :authenticate_api_v0_user!, only: [:create]
+  
   def index
     articles = Article.all
     render json: articles, each_serializer: Articles::IndexSerializer
   end
+
+  def create
+    if current_api_v0_user.role === 'research_group'
+      article = current_api_v0_user.articles.create!(article_params)
+      render json: { message: 'Article successfully created.' } 
+    else
+      render json: {}, status: 422
```

Comment:
Please do NOT respond with an empty object. Tell the client what's wrong and make use of the error response on the UI

Reply:
added error message to the response